### PR TITLE
Build UI docker image in github WF. Run the tests on the image

### PR DESCRIPTION
**Pull request by benjamin-bergia**

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -12,8 +12,8 @@
     "start": "craco start",
     "build": "craco build",
     "test:unit": "craco test",
-    "test": "yarn lint && yarn test:unit && yarn test:size",
     "test:size": "yarn build && bundlesize",
+    "test": "yarn lint && yarn test:unit --no-watch && yarn test:size",
```
Comment (drjova) [inline ui/package.json:16]: why do we need ``--no-watch``?

Comment (benjamin-bergia) [reply ui/package.json:16]: There is a limit on the number of files you can watch simultaneously and inside the docker container it seems to be lower. So the tests fail.

Comment (drjova) [reply ui/package.json:16]: @harunurhan will this affect local testing?

Comment (harunurhan) [reply ui/package.json:16]: no it won't

**Review by drjova — APPROVED** [review]

**Pull request merged**